py3status/modules/whatismyip.py

```diff
@@ -13,14 +14,13 @@
         (default 'ip')
     negative_cache_timeout: how often to check again when offline (default 2)
     timeout: how long before deciding we're offline (default 5)
-    url: change IP check url (must output a plain text IP address)
-        (default 'http://ultrabug.fr/py3status/whatismyip')
```

Comment:
I don't really care tbh. I created my service because all the ones I used to use were sloppy and unreliable in terms of availability. If you trust freegeoip, let's go for it